anker/src/processor.rs

```diff
@@ -533,6 +537,27 @@ fn process_change_token_swap_pool(
     anker.save(accounts.anker)
 }
 
+/// Change Anker's `sell_rewards_min_out_bps`.
+/// Solido's manager needs to sign the transaction.
+#[inline(never)]
+fn process_change_sell_rewards_min_out_bps(
+    program_id: &Pubkey,
+    accounts_raw: &[AccountInfo],
+    sell_rewards_min_out_bps: u64,
+) -> ProgramResult {
+    let accounts = ChangeSellRewardsMinBpsAccountsInfo::try_from_slice(accounts_raw)?;
+    let (solido, mut anker) = deserialize_anker(program_id, accounts.anker, accounts.solido)?;
+    solido.check_manager(accounts.manager)?;
+
+    // Cannot be greater than 100%.
+    if sell_rewards_min_out_bps > 1_000_000 {
```

Comment:
```suggestion
    if sell_rewards_min_out_bps > 10_000 {
```
Also here, it’s not basis points of a percent, it’s basis points.